app/javascript/packs/foremMobile.js

```diff
@@ -0,0 +1,114 @@
+/* global Runtime, fetchBaseData */
+
+function initializeNamespaceWhenPageIsReady() {
+  setTimeout(() => {
+    // Wait for data-loaded so we can ensure initializers have executed. This
+    // way we know the Runtime class will be available globally
+    if (document.body.getAttribute('data-loaded') === 'true') {
+      // We're ready to initialize
+      if (Runtime.currentMedium() === 'ForemWebView') {
+        loadForemMobileNamespace();
+      }
+    } else {
+      // Page hasn't initialized yet. We need to wait until the page is ready
+      initializeNamespaceWhenPageIsReady();
+    }
+  }, 100);
+}
+
+function loadForemMobileNamespace() {
```

Comment:
Ahh understood. I only bring it up because the whole pack file code will be loaded because of this in `application.html.erb`.

```
        <%= javascript_packs_with_chunks_tag "base", "foremMobile", "Search", "runtimeBanner", "onboardingRedirectCheck", "contentDisplayPolicy", defer: true %>
      <% else %>
        <%= javascript_packs_with_chunks_tag "base", "foremMobile", "Search", "runtimeBanner", defer: true %>
```

That pack is required so you can bootstrap for Forem mobile, but you can still dynamically import the `loadForemMobileNamespace()` function. Otherwise, that code is always loaded (not run) when a page loads.

Reply:
Ok I think I may have implemented this [on this last commit](https://github.com/forem/forem/pull/15212/commits/a97c1f2ec8fcfa66d44ddaae3060fb970f38c2bb), not 100% sure it's what it should be though 😅  

Let me know if that's the correct way to use the dynamic import. This webpack-package-module configs are uncharted territory for me 😄